src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.js

```diff
@@ -54,6 +55,7 @@ const UniversalNavbarExpanded = ({
   trackSecondaryCtaClick,
   links,
   estimateExperiment,
+  singleCta,
```

Comment:
```suggestion
  singleCta = {},
```
so you don't get any cannot read property href of undefined... errors